tests/src/Kernel/ValueObject/ImageTest.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_bootstrap_theme\Kernel\ValueObject;
+
+use Drupal\entity_test\Entity\EntityTest;
+use Drupal\field\Entity\FieldConfig;
+use Drupal\field\Entity\FieldStorageConfig;
+use Drupal\file\Entity\File;
+use Drupal\image\Entity\ImageStyle;
+use Drupal\oe_bootstrap_theme\ValueObject\ImageValueObject;
+use Drupal\Tests\token\Kernel\KernelTestBase;
+
+/**
+ * Test image value object with image field type.
+ */
+class ImageTest extends KernelTestBase {
```

Comment:
I talked with Francesco. The class name is copied from oe_theme, but we should change it to follow best practice:
ImageValueObjectTest.
Don't worry if this is the same name in Unit and Kernel when we add the unit test in the future, the namespace is enough to disambiguate.
We will backport these changes to oe_theme later.